Sources/ComposableCoreLocation/Interface.swift

```diff
@@ -303,6 +303,12 @@ public struct LocationManager {
   var requestWhenInUseAuthorization: (AnyHashable) -> Effect<Never, Never> = { _ in
     _unimplemented("requestWhenInUseAuthorization")
   }
+    
+  #if (compiler(>=5.3) && !(os(macOS) || targetEnvironment(macCatalyst))) || compiler(>=5.3.1)
+    var requestTemporaryFullAccuracyAuthorization: (AnyHashable, String) -> Effect<Never, Never> = { _, _  in
+      _unimplemented("requestTemporaryFullAccuracyAuthorization")
+    }
+  #endif
```

Comment:
I think we need to drop these `#if` conditions because when building for Xcode 11 (which we currently support 😬) this endpoint is omitted, which means invoking it from `requestTemporaryFullAccuracyAuthorization(id:purposeKey:)` fails to compile.

I don't have Xcode 11 installed to test, but I believe everything work if you just drop those compiler directives.

Reply:
I didn't realise! Thank you for the heads up.